logserver/logserver.c

```diff
@@ -868,7 +868,13 @@ void pv_logserver_toggle(struct pantavisor *pv, const char *running_rev)
 
 static void logserver_capture_dmesg()
 {
-	errno = 0;
+	if (logserver_utils_printk_devmsg_on() < 0)
+		pv_log(DEBUG,
+		       "could not set printk_devmsg=on, some messages could lose");
+	else
```

Comment:
I'd remove this and maybe other new log DEBUG traces you put there except the ones that indicate an error to make it less verbose. But if you think they are useful for device operation, we can keep them.